mozilla-release/browser/base/content/aboutDialog.xul

```diff
@@ -44,8 +44,6 @@
       <vbox id="rightBox" flex="1">
         <label id="distribution" class="text-blurb"/>
         <label id="distributionId" class="text-blurb"/>
-#expand <label id="version">&version.basedon;__MOZ_APP_VERSION__</label>
```

Comment:
deleting lines that is not covered by tests seems little bit risky. But as this one is ours it is ok.